Source/WebCore/Configurations/WebCore.xcconfig

```diff
@@ -26,7 +26,9 @@
 
 GCC_PREFIX_HEADER = WebCorePrefix.h;
 
-WK_USE_APPLEJPEGXL = NO;
+WK_USE_APPLEJPEGXL = $(WK_USE_APPLEJPEGXL_$(WK_PLATFORM_NAME));
+WK_USE_APPLEJPEGXL_macosx = $(WK_USE_APPLEJPEGXL$(WK_MACOS_1400));
+WK_USE_APPLEJPEGXL_MACOS_BEFORE_1400 = YES;
```

Comment:
It will have to be disabled in pre-Sonoma open source builds. We include the framework in STP for older OSes, but it's not present on the bots.

Reply:
Hm…it's checked in as a SDK addition in these paths:
```
WebKitLibraries/SDKs/macosx12.0-additions.sdk/System/Library/PrivateFrameworks/AppleJPEGXL.framework/AppleJPEGXL.tbd
WebKitLibraries/SDKs/macosx13.0-additions.sdk/System/Library/PrivateFrameworks/AppleJPEGXL.framework/AppleJPEGXL.tbd
```

If I remember correctly, the idea is that open-source builds should weak-link against it and handle it being missing at runtime. That link command doesn't have the SDKAdditions path at all, so maybe there is a search path missing.

Reply:
I stand corrected.